.github/ISSUE_TEMPLATE/1-bug-report.yml

```diff
@@ -1,11 +1,11 @@
 name: "\U0001F41B Bug report"
-description: Create a report to help us improve
+description: Report a issue that you have spotted
 body:
```

Comment:
Hi @ArjunSharda, thank you for the PR. 
Could you please change the  `Report a issue` to `Report an issue`